backend/leaderboard/tests/test_stats.py

```
@@ -357,7 +359,7 @@ def test_community_social_link_contributions_do_not_count_as_members_or_activity
         self.assertEqual(response.data['contribution_count'], 0)
         self.assertEqual(response.data['total_points'], link_points)
 
-    def test_builder_lookup_uses_real_builder_ranking_eligibility(self):
+    def test_builder_lookup_uses_accepted_submission_ranking_eligibility(self):
```

Ruff (0.15.20)

[warning] Too many statements (76 > 50)

(PLR0915)